docs/src/further_topics/ugrid/operations.rst

```diff
@@ -0,0 +1,791 @@
+.. _ugrid operations:
+
+Working with Mesh Data
+***********************
+
+.. note:: Several of the operations below rely on the optional dependencies
+          mentioned in :doc:`partner_packages`.
+
+.. list-table::
+    :align: left
+    :widths: 35, 75
+
+    * - `Making a Mesh`_
+      - |tagline: making a mesh|
+    * - `Making a Cube`_
+      - |tagline: making a cube|
+    * - `Save`_
+      - |tagline: save|
+    * - `Load`_
+      - |tagline: load|
+    * - `Plotting`_
+      - |tagline: plotting|
+    * - `Regional Extraction`_
+      - |tagline: regional extraction|
+    * - `Regridding`_
+      - |tagline: regridding|
+    * - `Equality`_
+      - |tagline: equality|
+    * - `Combining Cubes`_
+      - |tagline: combining cubes|
+    * - `Arithmetic`_
+      - |tagline: arithmetic|
+
+..
+    Below: use demo code over prose wherever workable. Headings aren't an
+     exhaustive list (can you think of any other popular operations?).
+
```

Comment:
All the way through the below, I would reduce the number of links out to the rest of the documentation. It may be that I read documentation oddly, but my tendency is to Ctrl-click out on each link so I can learn more in a minute. The repeated linking to "Cube" and "Mesh" makes this hard and breaks up readability. I'd suggest linking to things the first time that they're referenced in a given operation but not afterwards (when I'd just use the `stuff` quotes).
This would make it easier to spot when something new is referenced. It's not going to affect conceptual understanding though, so happy if you ignore this in the name of time.

Reply:
>ignore this in the name of time

I think I might have to, that's rather fiddly! It's an interesting question though, I've so far adopted the approach that **any** reference to the API should use a Sphinx domain, but perhaps others feel differently @SciTools/iris-devs?

Reply:
I tend to look out for this since I really struggled reading this page of the docs: https://scitools-iris.readthedocs.io/en/stable/developers_guide/contributing_getting_involved.html - I'd be interested to know it other people find something similar.

Reply:
>I tend to look out for this since I really struggled reading this page of the docs: https://scitools-iris.readthedocs.io/en/stable/developers_guide/contributing_getting_involved.html

FWIW I'm with you on that one.

I guess I must have a different subconscious contract when I see an API object - I see it mostly as formatting and don't feel the same obligation/compulsion to click as with a normal link.

Reply:
That's fair - it does help that `Cube` always looks like `Cube` rather than being replaced by other text that hides that it's the same link